cmd/clusterctl/README.md

```diff
@@ -4,6 +4,19 @@
 
 Read the [experience doc here](https://docs.google.com/document/d/1-sYb3EdkRga49nULH1kSwuQFf1o6GvAw_POrsNo5d8c/edit#). To gain viewing permissions, please join either the [kubernetes-dev](https://groups.google.com/forum/#!forum/kubernetes-dev) or [kubernetes-sig-cluster-lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle) google group.
 
+## Goals
```

Comment:
These should probably be in a proposal doc rather than a readme file.

Reply:
Or added as sub-section to https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/scope-and-objectives.md ?

Reply:
I'll defer that to @timothysc, since he is wrangling the `clusterctl` work currently.